services/movie.py

```diff
@@ -0,0 +1,36 @@
+from typing import Optional
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: Optional[list[int]] = None,
+        actors_ids: Optional[list[int]] = None) -> Movie:
+    queryset = Movie.objects.all()
+
+    if genres_ids is not None:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids is not None:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(movie_title: str,
+                 movie_description: str,
+                 genres_ids: Optional[list[int]] = None,
+                 actors_ids: Optional[list[int]] = None) -> Movie:
```

Comment:
invalid return typehint

Reply:
not fixed. The function doesn't return Movie.